catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/stage/StartListeningStageTest.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2020 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick.stage
+
+import android.Manifest
+import android.speech.tts.TextToSpeech
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.rule.GrantPermissionRule
+import org.catrobat.catroid.ProjectManager
+import org.catrobat.catroid.content.Project
+import org.catrobat.catroid.content.Sprite
+import org.catrobat.catroid.content.StartScript
+import org.catrobat.catroid.content.bricks.StartListeningBrick
+import org.catrobat.catroid.formulaeditor.UserVariable
+import org.catrobat.catroid.stage.StageActivity
+import org.catrobat.catroid.testsuites.annotations.Cat.CatrobatLanguage
+import org.catrobat.catroid.testsuites.annotations.Cat.SensorBox
+import org.catrobat.catroid.testsuites.annotations.Cat.SettingsAndPermissions
+import org.catrobat.catroid.testsuites.annotations.Level.Functional
+import org.catrobat.catroid.uiespresso.util.actions.CustomActions
+import org.catrobat.catroid.uiespresso.util.rules.BaseActivityTestRule
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import java.util.Locale
+
+class StartListeningStageTest {
+    private lateinit var project: Project
+    private lateinit var tts: TextToSpeech
```

Comment:
please change this variable to something longer like textToSpeech

Reply:
This makes no sense to me, tts is globally known as shortcut for text to speech!

Reply:
> please change this variable to something longer like textToSpeech

This makes no sense to me, tts is globally known as shortcut for text to speech!